src/ontology/components/relation.owl

```diff
@@ -74,6 +81,20 @@
     
 
 
+    <!-- http://purl.obolibrary.org/obo/VBO_0300020 -->
+
+    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/VBO_0300020">
+        <obo:IAO_0000115>A relation between a breed and the geographic location where the breed was reported to exist by country-nominated FAO National Coordinators.</obo:IAO_0000115>
+        <obo:IAO_0000233 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/209</obo:IAO_0000233>
+        <terms:contributor rdf:resource="https://orcid.org/0000-0002-4142-7153"/>
+        <terms:contributor rdf:resource="https://orcid.org/0000-0002-5002-8648"/>
+        <terms:created rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2024-11-05T18:13:40Z</terms:created>
+        <rdfs:comment>Officially nominated FAO National Coordinators report breeds existing in their country (these breeds are reported in the DAD-IS breed list). ‘of breed reported in&apos; refers to the region where the breed was reported. For example: VBO:0007427 represents the “Alatau” breed reported to be found in Kyrgyzstan by the FAO coordinators.</rdfs:comment>
+        <rdfs:label xml:lang="en">of breed reported in</rdfs:label>
```

Comment:
A property name should always to fully self contained, even if the sentences constructed by a person sounds odd. This usually means mentioning the domain:

```suggestion
        <rdfs:label xml:lang="en">breed reported in geographic location</rdfs:label>
```

Also feel free to make "breed" the domain of this relation, but that's not imperative here.